Neos.ContentRepositoryRegistry/Classes/Command/ContentCommandController.php

```diff
@@ -148,7 +148,8 @@ private function createVariantRecursivelyInternal(int $level, NodeAggregateId $p
 
         foreach ($childNodes as $childNode) {
             if ($childNode->classification->isRegular()) {
-                if ($childNode->nodeType->isOfType('Neos.Neos:Document')) {
+                $childNodeType = $contentRepository->getNodeTypeManager()->getNodeType($childNode->nodeTypeName);
+                if ($childNodeType->isOfType('Neos.Neos:Document')) {
```

Comment:
I'm wondering why do we need the `Neos.Neos:Document` here? Isn't it this a Neos thing? As `Neos.ContentRepositoryRegistry` has no dependency we can use the const of NodeTypeNameFactory unfortunately.

Reply:
Agreed, the Registry should not care about document nodes. Would you open a new issue for this, please?

Reply:
https://github.com/neos/neos-development-collection/issues/4488